auction_keeper/gas.py

```diff
@@ -64,7 +66,10 @@ def get_gas_price(self, time_elapsed: int) -> Optional[int]:
 
         # if API produces no price, or remote feed not configured, start with a fixed price
         if fast_price is None:
-            initial_price = self.fixed_gas if self.fixed_gas else 10 * self.GWEI
+            if self.fixed_gas:
+                initial_price = self.fixed_gas
+            else:
+                initial_price = max(self.web3.manager.request_blocking("eth_gasPrice", []), 1 * self.GWEI)
```

Comment:
would be better to externalize it in `pygasprice-client`

Reply:
I don't see value in introducing a Web3 dependency in `pygasprice-client`.  If anywhere, I could move this into `pymaker`.  Thoughts?

Reply:
`pymaker` should work

Reply:
Please review: https://github.com/makerdao/pymaker/pull/186

Reply:
Done; please review.